src/init1.cc

```diff
@@ -360,7 +358,7 @@ static cptr r_info_flags6[] =
 	"TELE_AWAY",
 	"TELE_LEVEL",
 	"DARKNESS",
-	"TRAPS",
+	"TRAPS",	/* Unused */
```

Comment:
Just "XXX" it. That way any left-over "TRAPS" flags in the r_info.txt file will yield a parse error. (These should be trivial to remove automatically now with a "grep -v ^S:TRAPS" command, btw.)